server/api/eliteApi.js

```diff
@@ -7,6 +7,10 @@ const eliteApiFactory = client => {
     contextProperties.setResponsePagination(context, response.headers)
     return response.data
   }
+  const processSuperAgentResponse = context => response => {
```

Comment:
will be able to replace `processResponse` with this one, once everything been converted